frontend/src/queries/useAccountQuery.ts

```diff
@@ -311,6 +311,7 @@ export const useAccountQuery = ({
 	const identifier = id?.value ? { id: id.value } : { address: address?.value }
 
 	const { data, fetching, error, executeQuery } = useQuery({
+		context: { url: useRuntimeConfig().public.apiUrlRust },
```

Comment:
We cannot migrate AccountRewardMetrics without migrating this.
 
The view seems to be all implemented.